impeller/entity/contents/content_context.cc

```diff
@@ -334,6 +334,24 @@ ContentContext::ContentContext(
         std::make_unique<ClipPipeline>(*context_, clip_pipeline_descriptor));
   }
 
+  glyph_atlas_pipelines_.CreateDefault(
+      *context_, options,
+      {static_cast<Scalar>(
+          GetContext()->GetCapabilities()->GetDefaultGlyphAtlasFormat() ==
+          PixelFormat::kA8UNormInt)});
+  texture_downsample_pipelines_.CreateDefault(*context_, options_trianglestrip);
+  rrect_blur_pipelines_.CreateDefault(*context_, options_trianglestrip);
+  texture_strict_src_pipelines_.CreateDefault(*context_, options);
+  tiled_texture_pipelines_.CreateDefault(*context_, options, {supports_decal});
+  gaussian_blur_pipelines_.CreateDefault(*context_, options_trianglestrip,
+                                         {supports_decal});
+  border_mask_blur_pipelines_.CreateDefault(*context_, options_trianglestrip);
+  color_matrix_color_filter_pipelines_.CreateDefault(*context_,
+                                                     options_trianglestrip);
+  porter_duff_blend_pipelines_.CreateDefault(*context_, options_trianglestrip,
+                                             {supports_decal});
+  vertices_uber_shader_.CreateDefault(*context_, options, {supports_decal});
+
```

Comment:
Can you put these in the brackets above?  That's what that section is for, higher priority pipelines.  Maybe I should have added a comment there last time I sorted these.

Reply:
Not all of these shaders should move up top though. The re-ordering of these is fine, but the glyph atlas one should move of

Reply:
Oh, is that what that was for. I'll move and add the comment.

Reply:
> but the glyph atlas one should move of

Elaborate?

Reply:
oops. "up" I mean. Like we should create glyph atlas after solid color et cetera.

Reply:
Oh, I found a few other cases like the rrect blur stuff in the gallery. So at least some of this necessary IMO.

Reply:
The changes you made are fine, I'm suggesting moving the glyph atlas up even higher.

Reply:
Derp. Ok ok. Got it.

Reply:
Done.